_config.yml

```diff
@@ -1,6 +1,27 @@
 title: Site title
 
+# GitHub information
+# This is used for adding an edit this page link to the footer
```

Comment:
are we going to put this in the footer later? If not, not sure we should add it.

Reply:
We can show the "edit this page" link by default, is that what you are suggesting?

Reply:
Yes! If we ask for the info in the config file, we should implement it or put the code in with a comment.